mainnet_contract/README.md

```diff
@@ -2,48 +2,36 @@
 This folder has provided scripts for:
 - Contract Building and Testing
 - Contract Deployment
-- API Interaction with Deployed Subnet Contract
 
 ## Contract Building and Testing:
-Make sure you have installed [TruffleSuite](https://trufflesuite.com/docs/truffle/how-to/install/) ahead.
+Environmental preparation
+###### Nodejs 16 or higher version
 
-We have provided the precompiled contracts in JSON format under `./build/contracts`. But if you want to rebuild it in local machine, you can do this under current folder:
+Install dependencies
+```
+ yarn
```

Comment:
space at the front

Reply:
Got it ,The build directory was replaced with artifacts and ignored in .gitignore, I can modify it if needed

Reply:
Or do you mean to put a little human introduction on the front?

Reply:
not... I mean you put a space at the front of yarn, ` yarn`, but it's not necessary right?

Reply:
😂 Sure, i will delete space